components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AccessTokenDAOImpl.java

```diff
@@ -351,6 +346,25 @@ private void insertAccessToken(String accessToken, String consumerKey, AccessTok
 
     }
 
+    private static String getInsertAccessTokenQuery() {
+
+        boolean isConsentedTokenColAvailable = OAuth2ServiceComponentHolder.isConsentedTokenColumnEnabled();
+        boolean isACRColAvailable = OAuth2ServiceComponentHolder.isAcrColumnEnabled();
```

Comment:
Can we get rid of these column checks

Reply:
Same as above https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1984#discussion_r1064555552